content/javascript/concepts/objects/terms/getOwnPropertyDescriptor/getOwnPropertyDescriptor.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: 'Object.getOwnPropertyDescriptor()' 
+Description: 'Enables access to full information of a property.' 
+Subjects: 
+  - 'Web Development'
+  - 'Computer Science'  
+Tags: 
+  - 'Objects'
+  - 'Methods'  
+CatalogContent: 
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`Object.getOwnPropertyDescriptor()`** method is a built-in object that enables access to full information about a property and returns a property descriptor of the given property if it exists on the object, otherwise, it returns `undefined`. It only works on its properties directly defined by the object itself, meaning it does not have access to inherited properties from the object's prototype chain.
```

Comment:
```suggestion
The **`.getOwnPropertyDescriptor()`** method is a built-in object that enables access to full information about a property and returns a property descriptor of the given property if it exists on the object, otherwise, it returns **undefined**. It only works on its properties directly defined by the object itself, meaning it does not have access to inherited properties from the object's prototype chain.
```